app/store/navigation_store.ts

```diff
@@ -5,9 +5,12 @@ import {BehaviorSubject} from 'rxjs';
 
 import type {AvailableScreens} from '@typings/screens/navigation';
 
+const OVERLAY_EXCEPTIONS = ['floating-banner-overlay'];
```

Comment:
0/5 This would be better if is a Set instead of an array

Reply:
@enahum agreed.  Set should be faster if we ever make this exception list bigger, which I don't think we will ever.  But it's also will ensure it's unique, and also don't care about the order.